go.mod

```diff
@@ -4,5 +4,5 @@ go 1.12
 
 require (
 	github.com/davecgh/go-spew v1.1.1
-	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
this should bump to the same version used by k8s.io/kubernetes (v2.4.0)

Reply:
2.3.0 regressed indent behavior, 2.4.0 fixed it

Reply:
Oh ok, that was not se easy...it seems that v2.4.0 reverted the default behaviour for long lines wrapping. It has been reverted here https://github.com/go-yaml/yaml/pull/670 with the promise to make it API configurable. I need more time to see how to fix my unit test with v2.4.0 and I will update that PR.

Reply:
I suspect a unit test that fails with 2.2.8 and passes with 2.3.0 is demonstrating the regression in 2.3.0 and we don't want it to pass

Reply:
At the same time, I want to show that the expected behaviour that we want to be configurable is indeed configureable; so dependent projects who needs it also know about that. 

I have just updated the test with a call to `FutureLineWrap` . What do you think? should we remove the "long line test case" ?